controllers/provider-alicloud/pkg/controller/worker/machines_test.go

```diff
@@ -18,12 +18,12 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	"github.com/gardener/gardener-extensions/pkg/controller/common"
```

Comment:
move this to the group of other `github.com/gardener/gardener-extensions/...` imports